### DISCUSSION TITLE
How do I access Windows settings, if possible?

### DISCUSSION THREAD

**Discussion by Hulk-Lover** [The-Wineskin-Project/WineskinServer#511] [Q&A] [answered]
So I'm playing Skyrim through Steam, and it's running mostly fine, but I have a problem with NPC audio. After some searching, I've narrowed it down to an issue with my surround sound settings because I'm using stereo headphones. I want to be able to fiddle around with 5.1 vs 7.1 audio settings, but that's not in Mac settings and is supposedly in Windows settings. How would I access that, if I even can?

**Comment by yochenhsieh** [comment, marked as answer]
Have you tried winecfg?  
Wineskin -> Advanced -> Tools 
<img width="415" alt="截圖 2024-01-03 下午3 10 06" src="https://github.com/Gcenx/WineskinServer/assets/1897377/af6e453d-b6ff-4586-a528-38e7d8363c70">

**Reply by Hulk-Lover** [reply]
That may work, I'll try that when I get home.

**Reply by Hulk-Lover** [reply]
That is what I was looking for though. I just didn't click the device. Thanks!
